app/assets/stylesheets/application.css.scss

```diff
@@ -36,13 +36,30 @@
   position: relative;
 }
 
+.trophy-case-container {
+  background-image: url('http://urlm.co/blog/wp-content/uploads/ipad23.jpg');
+  background-position: center center;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+  color: #ffffff;
+  height: 100vh;
+  position: relative;
+}
+
+.text-white {
+  color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.